### ISSUE TITLE
KVMemNN for KBQA

### ISSUE THREAD

**Issue by Joseph1314** [facebook/MemNN#20]
Excuse me , now I am applying the model of key-value memory networks to  the KBQA,
but my experiment only shows the 46% accuracy on the test-set,and the paper of Key-Value MemNN shows the 93% accuracy on KB task,
does it really get the so high accuracy on kbqa? 
In addition, I only train the model with the qa-dataset without any pre-train process,does it master?
 Thank you very much if any one can give me a  reply or guidance
